tests/helpers/images.ts

```diff
@@ -20,3 +21,46 @@ export const deleteAllImages = async (
   const notification = page.locator(".toast-notification");
   await expect(notification).toHaveText(/.*deleted.*/);
 };
+
+export const createImage = async (page: Page, instance: string) => {
+  //Creates an image from an instance
+  const imageAlias = `${instance}-alias`;
+  await visitInstance(page, instance);
+  await page.getByRole("button", { name: "Create Image" }).click();
+  page.getByRole("dialog", { name: "Create image from instance" });
```

Comment:
Is something missing from this line - maybe `expect(...).toBeVisible()`?

Reply:
I am not sure that would be much use within the createImageFromInstance function as a successful image is indicated by the toast notification, which we do check for. However it might be useful in getImageNameFromAlias.

Reply:
I was mainly confused why the line with `  page.getByRole("dialog", { name: "Create image from instance" });` is there after all. Maybe it can be removed?

Reply:
Oooh, right I see. Yep, that is understandable, that can be removed.